plugins/modules/xenserver_guest_info.py

```diff
@@ -8,48 +8,46 @@
 from __future__ import absolute_import, division, print_function
 __metaclass__ = type
 
-DOCUMENTATION = r'''
----
+DOCUMENTATION = r"""
 module: xenserver_guest_info
 short_description: Gathers information for virtual machines running on Citrix Hypervisor/XenServer host or pool
-description: >
-   This module can be used to gather essential VM facts.
+description: This module can be used to gather essential VM facts.
 author:
-- Bojan Vitnik (@bvitnik) <bvitnik@mainstream.rs>
+  - Bojan Vitnik (@bvitnik) <bvitnik@mainstream.rs>
 notes:
-- Minimal supported version of XenServer is 5.6.
-- Module was tested with XenServer 6.5, 7.1, 7.2, 7.6, Citrix Hypervisor 8.0, XCP-ng 7.6 and 8.0.
-- 'To acquire XenAPI Python library, just run C(pip install XenAPI) on your Ansible Control Node. The library can also be found inside
-   Citrix Hypervisor/XenServer SDK (downloadable from Citrix website). Copy the XenAPI.py file from the SDK to your Python site-packages on your
-   Ansible Control Node to use it. Latest version of the library can also be acquired from GitHub:
-   U(https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py)'
-- 'If no scheme is specified in C(hostname), module defaults to C(http://) because C(https://) is problematic in most setups. Make sure you are
-   accessing XenServer host in trusted environment or use C(https://) scheme explicitly.'
-- 'To use C(https://) scheme for C(hostname) you have to either import host certificate to your OS certificate store or use C(validate_certs: no)
-   which requires XenAPI library from XenServer 7.2 SDK or newer and Python 2.7.9 or newer.'
+  - Minimal supported version of XenServer is 5.6.
+  - Module was tested with XenServer 6.5, 7.1, 7.2, 7.6, Citrix Hypervisor 8.0, XCP-ng 7.6 and 8.0.
+  - 'To acquire XenAPI Python library, just run C(pip install XenAPI) on your Ansible Control Node. The library can also be found inside Citrix
+    Hypervisor/XenServer SDK (downloadable from Citrix website). Copy the C(XenAPI.py) file from the SDK to your Python site-packages on your Ansible
+    Control Node to use it. Latest version of the library can also be acquired from GitHub:
+    U(https://raw.githubusercontent.com/xapi-project/xen-api/master/scripts/examples/python/XenAPI/XenAPI.py)'
+  - 'If no scheme is specified in C(hostname), module defaults to C(http://) because C(https://) is problematic in most setups. Make sure you
+    are accessing XenServer host in trusted environment or use C(https://) scheme explicitly.'
+  - 'To use C(https://) scheme for C(hostname) you have to either import host certificate to your OS certificate store or use O(validate_certs=no) which
+    requires XenAPI library from XenServer 7.2 SDK or newer and Python 2.7.9 or newer.'
 requirements:
-- XenAPI
+  - XenAPI
```

Comment:
@bvitnik also, this block is repeated in all but one of the modules. I would suggest transofrming it into a doc fragment, to make it DRY.

Reply:
This is a good idea now that I look at it. My initial idea was probably to keep notes specific to the module because there was a plan to implement a slew of other modules that never came into being. Each module was expected to have slightly different notes. Also, the three modules mentioned here were originally a single module and were split into three at later time (copy-paste job). Since there are some updated to be made on the notes anyway, I will consider moving them to doc fragment.